### DISCUSSION TITLE
How to replicate a Postman POST request in Mirth

### DISCUSSION THREAD

**Discussion by BondAcropolis** [nextgenhealthcare/connect#6228] [Q&A]
Hi everyone,

I'm trying to replicate a POST request in Mirth that I successfully sent using Postman. Here are the details of the request:

Request:
`POST /qatic/APIRIUUrgencias/agregarAtencion HTTP/1.1
Host: api.minsal.cl
Authorization: Bearer xxxxxxxxxxx
Content-Length: 2293
Content-Type: multipart/form-data; boundary=----WebKitFormBoundary7MA4YWxkTrZu0gW

------WebKitFormBoundary7MA4YWxkTrZu0gW
Content-Disposition: form-data; name="atencion"

{
    "nombreSolucion" : "RIU Urgencias - Pruebas",
    "numeroProceso" : null,
    "mesAtencion" : 2,
    "anoAtencion" : 2023,
    "codigoSS" : "23",
    "codigoEstablecimiento" : 114104,
    "idDAU" : "PVC09o00q3q52109",
    "idBDPersonas" : "CL9876",
    "idPaciente" : "PAC9876",
    "run" : "9999999",
    "dv" : "9",
    "horaAdmision" : "23:45",
    "procedenciaPaciente" : 2,
    "unidadAtencion" : "02",
    "motivoConsulta" : "DOLOR ABDOMINAL",
    "clasificacionConsulta" : "05"

}
------WebKitFormBoundary7MA4YWxkTrZu0gW--

`I'm unsure how to set up the same request in Mirth, especially the multipart/form-data aspect with the boundary. Any guidance or examples on how to achieve this would be greatly appreciated.

Thank you!

**Comment by pacmano1** [comment]
https://forums.mirthproject.io/forum/mirth-connect/support/18273-multipart-form-post-of-base64-encoded-pdf-solved, I think this may be what you want, but a link to the API documentation would be useful.
